File: WebSolution/PMSNGApp/src/app/time-sheet/models/TimeSheetEntry.ts
import { Activity } from "src/app/activity/Models/Activity";

export class TimeSheetEntry{
    constructor(
        public id:number,
        public activityId:number,
        public fromTime:string,
        public toTime:string,
        public timeSheetId:number,

        public durationInMinutes:number,
        public durationInHours:string,
        ){
    }

    public activity:Activity|undefined;


        
 
}

File: Solution/WebSolution/Monolithic/Models/TimesheetEntry.cs
using System;

namespace TFLPortal.Models;


public class TimesheetEntry
{
   public int Id { get; set; }
   public TimeOnly FromTime { get; set; }
   public TimeOnly ToTime { get; set; }
   public int TimesheetId { get; set; }
   public int TaskId { get; set; } 

   public double Hours {get;set;}
   
}
